test/functional/mempool_updatefromblock.py

```diff
@@ -97,6 +98,53 @@ def transaction_graph_test(self, size, n_tx_to_mine=None, fee=100_000):
             assert_equal(entry['ancestorcount'], k + 1)
             assert_equal(entry['ancestorsize'], sum(tx_size[0:(k + 1)]))
 
+        self.generate(self.nodes[0], 1)
+        assert_equal(self.nodes[0].getrawmempool(), [])
+        wallet.rescan_utxos()
+
+        # Make 200 outputs to spend via wallet
+        wallet.send_self_transfer_multi(from_node=self.nodes[0], num_outputs=200)
```

Comment:
MiniWallet doesn't appear to support more than 49 utxos at a time? Unclear to me what's going on so I did this instead.

Reply:
Not exactly sure if this is what you are asking, but: new blocks with coinbase transaction outputs for MiniWallet can be generated by passing an instance of the latter to `self.generate` (instead of the node object), i.e. in this case:
```
        self.generate(wallet, 300)
```
(added +100 to let the coins mature)

Reply:
Oh, mine to wallet, not to node which has attached wallet :facepalm: